tests/functional/helmholtz/kh_2d.py

```
@@ -0,0 +1,238 @@
+#!/usr/bin/env python3
+
+import os
+import numpy as np
+
+import pyphare.pharein as ph
+from pyphare.cpp import cpp_lib
+from pyphare.simulator.simulator import Simulator
```

Ruff (0.8.2)

`pyphare.simulator.simulator.Simulator` imported but unused

Remove unused import: `pyphare.simulator.simulator.Simulator`

(F401)